ctr/checkpoint.go

```diff
@@ -107,16 +111,22 @@ var createCheckpointCommand = cli.Command{
 			fatal("checkpoint name cannot be empty", ExitStatusMissingArg)
 		}
 		c := getClient(context)
+		checkpoint := types.Checkpoint{
+			Name:        name,
+			Exit:        context.Bool("exit"),
+			Tcp:         context.Bool("tcp"),
+			Shell:       context.Bool("shell"),
+			UnixSockets: context.Bool("unix-sockets"),
+		}
+
+		for _, ns := range context.StringSlice("empty-ns") {
```

Comment:
Even better then :). I'd still create a temporary variable for readability though.